### ISSUE TITLE
ellipsize

### ISSUE THREAD

**Issue by MarcusWolschon** [mikepenz/MaterialDrawer#2724]
There seems to be no  way to set the "ellipsize" attribute for a PrimaryDrawerItem.
The prevents an implementation of elipsyze="middle" for long folder names in [K9
Issue 5290](https://github.com/k9mail/k-9/issues/5290)

**Comment by MarcusWolschon** [comment]
The ability to modify android:lines and android:singleLine would also be benefitial.

**Comment by mikepenz** [comment]
@MarcusWolschon the `MaterialDrawer` allows you to also create custom drawer items if you need more features which are not covered by the default items. 

Just looked at the referenced issue. And I see you solved it. 

It may make sense to add support for this.

**Comment by MarcusWolschon** [comment]
I made an ugly, temporary workaround for K-9 until a better way is decided on. Not really a fix.
I tried a custom layout and referenced it in a subclass with overloaded property for the layout ID
but it would always load the default layout.

Given that it's dependent on what kind of text is displayed, I figured adding it as an exposed property upstream made the most sense.
I have another issue to work on next, so I can't make a patch and a Merge Request myself right away.
